vita3k/gui/src/controls_dialog.cpp

```diff
@@ -104,6 +105,7 @@ static void remapper_button(GuiState &gui, EmuEnvState &emuenv, int *button, con
         std::array<int, total_key_entries> original_state;
         prepare_map_array(emuenv, original_state);
         while (gui.is_capturing_keys) {
+            emuenv.ctrl.drop_inputs = false; // set ctrl.drop_inputs to false, so they don't get discarded while rebinding
```

Comment:
i no understund here ?

Reply:
If you remove this line, Vita3K locks up when you input a keyboard bind.

Reply:
Want said your code is wrong here
Looking your code inside interface

Reply:
Remove it here

Reply:
Your current code is full wrong, you set truc in end for set false here, is so never on truc is no sense
Juste done what i said in interface, and delete this

Reply:
you no understund why your code here have no sense..
you get not responding here because you r code inside interface is on wrong place
here you set code to false for can passe and captured key, have no sense, set it true for set it false in same time

Move code in interface wjere i said, and delete this line and works fine

Reply:
No. Your solution has zero sense to execute more code than necessary. I understand I am arguing over a cycle or two, but my solution is valid and your solution arguably makes it less readable. I have asked two other people if your solution is good, and they have both said no. I am sick of arguing.

Reply:
you litteraly set one value to true for ste to false here... is you execute more code are needed compare me
is not my solution, is just logical